app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughActivityPresenter.kt

```diff
@@ -138,4 +138,12 @@ class WalkthroughActivityPresenter @Inject constructor(
   fun setTopicId(topicId: String) {
     this.topicId = topicId
   }
+
+  fun hideProgressBarAndShowHeader() {
+    binding.viewModel?.hideProgressBarAndShowHeader()
```

Comment:
Creating a global viewModel var and using it here will be helpful rather than accessing using binding. 

```
private val walkthroughViewModel by lazy {
    getWalkthroughViewModel()
  }
```

Reply:
Like other places in the file, I have used getWalkthroughViewModel directly, because then we face that JVM same signature error.

Reply:
I didn't get this? 
What error you are mentioning exactly?

Reply:
@anandwana001 
```
Platform declaration clash: The following declarations have the same JVM signature (getWalkthroughViewModel()Lorg/oppia/android/app/walkthrough/WalkthroughViewModel;): 
private final fun <get-walkthroughViewModel>(): WalkthroughViewModel defined in org.oppia.android.app.walkthrough.WalkthroughActivityPresenter
private final fun getWalkthroughViewModel(): WalkthroughViewModel defined in org.oppia.android.app.walkthrough.WalkthroughActivityPresenter
``
![Screenshot from 2020-10-07 18-25-47](https://user-images.githubusercontent.com/54636525/95333602-9ddf3880-08ca-11eb-80d8-601e7c1241fc.png)

Reply:
And I also tried [this](https://stackoverflow.com/questions/45510273/getting-platform-declaration-clash-when-using-interface-in-kotlin?rq=1) but it didn't help much

Reply:
Ah, you can change var name, I guess it should work then.